File: 2_foundation_of_mathematics_for_algorithm/ruby/code_003.rb
# https://atcoder.jp/contests/math-and-algorithm/tasks/math_and_algorithm_c

N = gets.to_i
exit unless 1 <= N && N <= 50

A = gets.chomp.split.map(&:to_i)
exit unless A.all? { |x| (1..100).include?(x) }

puts A.sum

File: 2_foundation_of_mathematics_for_algorithm/python/code_003.py
# https://atcoder.jp/contests/math-and-algorithm/tasks/math_and_algorithm_c

import sys

N = int(input())
if not 1 <= N <= 50:
    sys.exit()

A = list(map(int, input().split()))
if not len(A) == N or not all(1 <= N <= 100 for x in A):
    sys.exit()

Answer = 0

for item in A:
    Answer += item

print(Answer)
